src/cli.spec.js

```diff
@@ -9,11 +9,39 @@ const { setUp, tearDown } = require("./test-helpers.js");
 chai.use(require("chai-as-promised"));
 
 describe("CLI", function () {
+  // Moch the catalog validation schema
```

Comment:
```suggestion
  // Mock the catalog validation schema
```

Reply:
Fixed in 7b3428af9bfd65407dfceebcb7e6bd12de6de600